runhouse/main.py

```diff
@@ -590,159 +592,149 @@ def status(
 
-    if not sky_clusters and not clusters_in_den:
-        console.print("No existing clusters.")
+def _add_clusters_to_output_table(
+    table: Table, clusters: list[Dict], filters_requested: dict = None
+):
+    for rh_cluster in clusters:
+        last_active_at = rh_cluster.get("Last Active (UTC)")
+        last_active_at_no_offset = str(last_active_at).split("+")[
+            0
+        ]  # The split is required to remove the offset (according to UTC)
+        rh_cluster["Last Active (UTC)"] = last_active_at_no_offset
+        rh_cluster["Status"] = StatusColors.get_status_color(rh_cluster.get("Status"))
 
-    if sky_clusters:
-        # getting the on-demand clusters that are not saved in den.
-        sky_clusters = [
-            cluster
-            for cluster in sky_clusters
-            if f'/{rns_client.username}/{cluster.get("name")}'
-            not in clusters_in_den_names
-        ]
+        table = _add_cluster_as_table_row(table, rh_cluster)
 
-    total_clusters = len(clusters_in_den)
-    table_title = f"[bold cyan]Clusters for {rns_client.username} (Total: {total_clusters})[/bold cyan]"
 
-    table = Table(title=table_title, title_justify="left")
+def _print_msg_live_sky_clusters(sky_clusters: int):
+    if sky_clusters > 0:
+        console.print(
+            f"There are {sky_clusters} live cluster(s) that are not saved in Den. For more information, please run [bold italic]sky status -r[/bold italic]."
+        )
 
-    # Add columns to the table
-    table.add_column("Name", justify="left", no_wrap=True)
-    table.add_column("Cluster Type", justify="center", no_wrap=True)
-    table.add_column("Status", justify="left")
 
-    running_clusters = []
-    not_running_clusters = []
+def supported_time_units():
+    return ", ".join(list(TIME_UNITS.keys()))
 
-    for den_cluster in clusters_in_den:
-        # get just name, not full rns address. reset is used so the name will be printed all in white.
-        cluster_name = f'[reset]{den_cluster.get("name").split("/")[-1]}'
-        cluster_type = den_cluster.get("data").get("resource_subtype")
-        cluster_status = (
-            den_cluster.get("status") if den_cluster.get("status") else None
-        )
 
-        # currently relying on status pings to den as a sign of cluster activity.
-        # The split is required to remove milliseconds and the offset (according to UTC) from the timestamp.
-        # (status_last_checked is in the following format: YYYY-MM-DD HH:MM:SS.ssssss±HH:MM)
+@cluster_app.command("list")
+def cluster_list(
+    show_all: bool = typer.Option(
+        False,
+        "--a",
+        "--all",
+        help="Get all den clusters. Up to 50 most recently active clusters will be displayed.",
+    ),
+    since: Optional[str] = typer.Option(
+        None,
+        "--since",
+        help="Time duration to filter on. Minimum allowable filter is 1 minute. If the provided value will be less "
```

Comment:
```suggestion
        help="Time duration to filter on.  "
```
i think its duplicative since we already say the min allowable is 1 minute